village/src/components/Nav.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```

Comment:
good deformation of the named imports for the react-router-dom. Could you explain why this is better in 1 line instead of spreading it in to several import calls?